CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 10.1.1 "tin-hat" (2020-07-27)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **a11y:** interactivity checker detecting tabbable state incorrectly for videos, audio and object elements ([2825a9b](https://github.com/angular/components/commit/2825a9b)) |
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **datepicker:** emitting dateChange event when opposite range input changes ([#19995](https://github.com/angular/components/issues/19995)) ([d383bba](https://github.com/angular/components/commit/d383bba)), closes [#19968](https://github.com/angular/components/issues/19968) |
+| bug fix |  **drag-drop:** warn if connected container ID doesn't exist ([#20057](https://github.com/angular/components/issues/20057)) ([4fa9eb9](https://github.com/angular/components/commit/4fa9eb9)), closes [#20056](https://github.com/angular/components/issues/20056) |
+| bug fix |  **select:** indicating programmatic value change as user interaction in some cases ([#19970](https://github.com/angular/components/issues/19970)) ([88d8416](https://github.com/angular/components/commit/88d8416)), closes [#19967](https://github.com/angular/components/issues/19967) |
+| bug fix |  tabindex inputs not supporting coercion ([#20037](https://github.com/angular/components/issues/20037)) ([ec5986c](https://github.com/angular/components/commit/ec5986c)), closes [#20026](https://github.com/angular/components/issues/20026) |
+| bug fix |  **datepicker:** not updating calendar state if input changes while open ([#19973](https://github.com/angular/components/issues/19973)) ([8b48a9b](https://github.com/angular/components/commit/8b48a9b)), closes [#19959](https://github.com/angular/components/issues/19959) |
+| bug fix |  **drag-drop:** not transferring input value when cloning element for preview ([#20009](https://github.com/angular/components/issues/20009)) ([9833eea](https://github.com/angular/components/commit/9833eea)), closes [#19905](https://github.com/angular/components/issues/19905) |
+| bug fix |  **menu:** throw error if menu contains its own trigger ([#19943](https://github.com/angular/components/issues/19943)) ([16e8376](https://github.com/angular/components/commit/16e8376)), closes [#19941](https://github.com/angular/components/issues/19941) |
+| bug fix |  **scrolling:** viewport ruler events being run inside zone ([#15814](https://github.com/angular/components/issues/15814)) ([200ca35](https://github.com/angular/components/commit/200ca35)), closes [#18471](https://github.com/angular/components/issues/18471) |
```

Comment:
Should be under the CDK as well.

Reply:
The scrolling bugfix?

Reply:
Yep, sorry, I probably clicked the wrong cell.